<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog_parameters</class>
 <widget class="QDialog" name="dialog_parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QPushButton" name="bClose">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>99</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="lClick">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Click
ports</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lClick_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>120</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Mix
strategy</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bAuto">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPreset">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Preset
10 loops</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPresetPlus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPresetMinus">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bCaptureDevice">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Capture
devices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPlaybackDevice">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Playback
devices</string>
   </property>
  </widget>
  <widget class="QLabel" name="lClick_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lClick_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>111</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Controls</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bNewControl">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Create
new</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bControlList">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>310</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>View
list</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bMidiDevice">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>MIDI
capture
devices</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bMidiDevice_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>MIDI
playback
devices</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
